SDC-seed/ArangoDB/batchMaker.js

```diff
@@ -0,0 +1,23 @@
+const fs = require('fs');
+const writeListing = require('./dataGeneration.js');
+
+const writeBatches = (fileName, start, stop) => {
+  const writeListingJson = fs.createWriteStream(fileName);
+  writeListingJson.write('[\n', 'utf8');
+
+  writeListing(writeListingJson, 'utf-8', () => {
+    writeListingJson.write(']');
+    writeListingJson.end();
+  }, start, stop);
+};
+
+writeBatches('./dataFiles/listing0to1M.json', 0, 1000000);
```

Comment:
I really like how organized this file is and how you separated the writing process from the data generation process. One way to slim this file down even more could be to add the start/stop numbers as tuples to an array like this:
- [[0, 1000000],[1000000, 2000000] ...etc]

And then iterate through the array of tuples, calling the writeBatches function on index 0 and index 1 each time. That way you'd only have to have the function call on one line inside the loop. Also would need some string interpolation for the ranges in the file names, but you get the idea.